stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -64,3 +68,19 @@ export async function updateSecurity(securityId, security) {
   }
   throw new ValidationError(json.messages);
 }
+
+export async function deleteSecurity(securityId) {
+  const fetchOptions = {
+    method: "DELETE"
+  };
+  const options = await getWindowOptions();
+  const response = await fetch(
+    `${options.customData.apiBaseUrl}/securities-v2/${securityId}`,
+    fetchOptions
+  );
+  if (response.ok) {
+    return { message: "Security Successfully Deleted" };
```

Comment:
I don't think you should return a message here yourself, especially considering that you don't for any of the other service messages. Maybe just return nothing, which would indicate success still but it wouldn't resolve with a value (I think).

Reply:
Okay sure, if I want to display a message to the user should I set it in the components where it is used? Luke recently changed this service to not return a message so if I want to display one it will need to be hard coded either in the service or the components.

Reply:
It should only be the purpose of the service to call the API and return/resolve back what the API responded with. If the API has an empty response then the service should return/resolve with nothing. Displaying a message to the user based on that success or response should be the responsibility of the UI so I would say the components. There are occasions in non-restful APIs where messages can be returned though but in this case I think it should be up to the component.

Reply:
Okay great, that's what I thought so it is now done this way. Thanks for clarifying